diagnostic.md

```diff
@@ -54,7 +73,14 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big {}
+.contentPane {}
+.important,red {}
```

Comment:
The `,` says apply the below styling to both classes. in order to apply the styling to ELEMENTs that have BOTH the classes, its `.important.red`